src/main/resources/view/StudySpotListCard.fxml

```diff
@@ -61,8 +61,8 @@
         </HBox>
         <Region VBox.vgrow="ALWAYS" /> <!-- spacer -->
         <HBox spacing="4" alignment="CENTER_RIGHT">
```

Comment:
You can change the alignment here to `BASELINE_RIGHT` if you want the number and hours to be bottom-aligned.